### PR TITLE
fix: basic details fix

### PR DESCRIPTION
**Pull request by Drish-xD**
Fix Name and platform fields in basic details

### CODE REVIEW

**Review by suryabulusu — APPROVED** [review]

File: src/app/session/[type]/Steps/helper.ts [hunk 1]
```diff
@@ -11,64 +11,42 @@ import {
...
-        // signup_form: true,
-        // signup_form_id:
-        //   Number(apiOptions.signupForm?.find((item) => item.label.includes('Haryana'))?.value) ??
-        //   null,
-        id_generation: false,
-        redirection: false,
-        popup_form: true,
-        popup_form_id:
+        isIdGeneration: false,
+        isRedirection: false,
```
Comment (suryabulusu) [inline src/app/session/[type]/Steps/helper.ts:22]: redirection true

File: src/app/session/[type]/Steps/helper.ts [hunk 2]
```diff
@@ -11,64 +11,42 @@ import {
...
+        isPopupForm: true,
```
Comment (suryabulusu) [inline src/app/session/[type]/Steps/helper.ts:23]: popup form false

File: src/app/session/[type]/Steps/helper.ts [hunk 3]
```diff
@@ -11,64 +11,42 @@ import {
...
+        signupFormId:
           Number(apiOptions.signupForm?.find((item) => item.label.includes('Enable'))?.value) ??
           null,
-        id_generation: true,
-        redirection: true,
-        popup_form: true,
-        popup_form_id:
+        isIdGeneration: true,
+        isRedirection: true,
+        isPopupForm: true,
```
Comment (suryabulusu) [inline src/app/session/[type]/Steps/helper.ts:43]: false

File: src/app/session/[type]/Steps/helper.ts [hunk 4]
```diff
@@ -11,64 +11,42 @@ import {
...
-  value: string,
-  apiOptions: ApiFormOptions,
-  updateFormData: (data: Session | ((prevState: Session) => Session)) => void
-) => {
+export const setGroupPreset = (value: string, form: UseFormReturn, apiOptions: ApiFormOptions) => {
   if (!value) return;
 
-  let newDefaultData: Session = {};
+  let newDefaultData: Partial<basicFields> = {};
   switch (value) {
```
Comment (suryabulusu) [inline src/app/session/[type]/Steps/helper.ts:18]: have a default switch case... if its none of the values.. then 
redirection true
signup false
signin session type
popup false
form names null

**Review by suryabulusu — COMMENTED** [review]
left comments

**Comment by Drish-xD** [comment]
I have already set the defaults
- isRedirection = true
- isIdGeneration = false
- activate signUp = false
- isPopup = false


and now we are setting only authtype & SessionType using preset, as `sign-in` and `ID or IDDOB`

can we just skip(delete) preset and just set the defaults for these 2 fields also as `sign-in` and `ID`.

**Review by suryabulusu — APPROVED** [review]

**Pull request merged**